warehouse/router/upload.go

```diff
@@ -942,7 +942,20 @@ func (job *UploadJob) DTO() *model.UploadJob {
 }
 
 func (job *UploadJob) GetLocalSchema(ctx context.Context) (model.Schema, error) {
-	return job.schemaHandle.GetLocalSchema(ctx)
+	schemaRepo := repo.NewWHSchemas(job.db)
```

Comment:
Instead of creating everytime, can we handle it like we handle it for other repos like `uploadsRepo` in `UploadJobFactory`

Reply:
Added a `whSchemaRepo` field in the `UploadJob` struct